src/tools/dev/scripts/bv_support/bv_cgns.sh

```diff
@@ -393,18 +393,17 @@ function build_cgns
 
     # Disable fortran
     FORTRANARGS="--with-fortran=no"
-
     set -x
-    if [[ "$OPSYS" == "Darwin" ]] ; then
-        env CXX="$CXX_COMPILER" CC="$C_COMPILER" \
-            CFLAGS="$CFLAGS $C_OPT_FLAGS" CXXFLAGS="$CXXFLAGS $CXX_OPT_FLAGS" \
-            LDFLAGS="$LDFLAGS_ENV" LIBS="$LIBS_ENV" \
-            ./configure --enable-64bit --enable-cgnstools=no ${cf_build_type} $H5ARGS $FORTRANARGS --prefix="$VISITDIR/cgns/$CGNS_VERSION/$VISITARCH"
-    else
-        env CXX="$CXX_COMPILER" CC="$C_COMPILER" \
-            CFLAGS="$CFLAGS $C_OPT_FLAGS" CXXFLAGS="$CXXFLAGS $CXX_OPT_FLAGS" \
-            ./configure --enable-64bit --enable-cgnstools=no ${cf_build_type} $H5ARGS $FORTRANARGS --prefix="$VISITDIR/cgns/$CGNS_VERSION/$VISITARCH"
-    fi
+
+    info "    env CXX=\"$CXX_COMPILER\" CC=\"$C_COMPILER\" \
```

Comment:
This change should not be make. There were two changes to this file after #5725:
#5888 and #16980